pages/career.page.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+import JobCard from "../components/JobCard";
+import dynamic from "next/dynamic";
+import { useJobContext } from "../context/JobContext";
+import { Section } from "../components/UI";
+const Header = dynamic(() => import("../components/layout/Header"));
+
+const Career = () => {
+  const { jobDetails } = useJobContext();
+
+  return (
+    <>
+      <Header classname="tw-mb-10" />
+      <Section
+        kicker="Join Our Team"
+        heading="Open Positions"
+        subheading="Discover a Place Where Your Passion and Skills Drive Success"
+        align="center"
+        className="tw-mt-10 tw-pt-12 tw-pb-0 tw-mb-0"
+      />
+      <main className="tw-flex tw-flex-col tw-items-center tw-justify-center tw-px-4 tw-bg-[#f1f5f9]">
+        <div className=" tw-font-sans tw-w-full tw-max-w-6xl tw-mx-auto tw-text-center">
+          {Array.isArray(jobDetails) && jobDetails.length > 0 ? (
```

Comment:
in case of jobDetails is array of null items.. will it work?

Reply:
I changed it for better readability